pubsublite/internal/wire/settings.go

```diff
@@ -77,6 +81,7 @@ var DefaultPublishSettings = PublishSettings{
 	// By default set to a high limit that is not likely to occur, but prevents
 	// OOM errors in clients.
 	BufferedByteLimit: 1 << 30, // 1 GiB
+	ConfigPollPeriod:  10 * time.Minute,
```

Comment:
This feels a little long; did you chat with Evan to figure out what the right period is? I'm worried that a user would expect a faster reaction from their clients.

Reply:
@palmere-google 

Yes, Evan said he'll set it to 10 min and I see that it has been for the Java client lib: https://github.com/googleapis/java-pubsublite/pull/387.

I remember in the design doc that the duration was somewhat influenced by the read-only admin operations quota limit. I implemented retry with backoff for the ResourceExhausted code in Go to mitigate this.

Reply:
Sorry, I missed this

Yeah, I think with our current admin read limits, ten minutes is a good place to start. We can make the period smaller, but if we do, we should probably consider increasing the default admin read quota

Reply:
Ok thanks, I'll leave the default to 10 min for now.